crates/turbopack-core/src/resolve/parse.rs

```diff
@@ -116,10 +119,15 @@ impl Request {
                         }
                     }
                     if let Some(caps) = MODULE_PATH.captures(r) {
-                        if let (Some(module), Some(path)) = (caps.get(1), caps.get(2)) {
+                        if let (Some(module), Some(path), query) =
+                            (caps.get(1), caps.get(2), caps.get(3))
+                        {
                             return Request::Module {
                                 module: module.as_str().to_string(),
                                 path: path.as_str().to_string().into(),
+                                query: QueryMapVc::cell(query.map(|q| {
```

Comment:
Nit: Can this be done later? Eg, instead of splitting the `path` and `query` now, could this be done only when the import actually happens?